packages/amplify_core/lib/src/config/auth/cognito/user_pool.dart

```diff
@@ -17,6 +19,29 @@ class CognitoUserPoolConfig
     this.endpoint,
   });
 
+  @internal
+  factory CognitoUserPoolConfig.fromAuthOutputs(AuthOutputs authOutputs) {
```

Comment:
P.S. this used in `AuthConfiguration.fromAmplifyOutputs`. AuthConfiguration is an internal type used in Auth State Machine and is differnt from AuthConfig.